controller/search_blackbox_test.go

```diff
@@ -930,3 +930,85 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestFilterAssigneeNullAfterWIUpdate() {
```

Comment:
why do you add these tests in the `controller/search_blackbox_test.go` file ? why not at the repository level or at the work item controller level instead ? My point is, you're not using the SearchController in your test.

Reply:
@xcoulon I think he is [here](https://github.com/fabric8-services/fabric8-wit/pull/1699/files#diff-a2bd8cd90ce50fc5463d8b3539620c92R941), isn't he?

Reply:
I will write some tests in repo level.

Reply:
Fixed dfb5f849d2b6b07cc94dc1ce1dd3a706075e44ab